felix/wireguard/wireguard.go

```diff
@@ -1836,3 +1849,10 @@ func writeProcSys(path, value string) error {
 	}
 	return nil
 }
+
+func boolToBinaryString(input bool) string {
```

Comment:
Probably better to export this and use it in the fv test as well (instead of having a duplicate code). I think this function could be used in other places as well.

Reply:
@jrcichra it's not a good practice to import utility functions from fv/util to non-test codes. I tried to find a good place for it, but it seems not possible. Sorry for the previous comment, but please revert the change to the original one.